src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+import java.util.stream.Stream;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int SECOND_ELEMENT = 1;
+    private static final String TXT_SUPPLY = "supply";
+    private static final String TXT_BUY = "buy";
+    private static final String TXT_RESULT = "result";
+    private static final String TXT_SEPARATOR = ",";
+
+    private void appendStringBuilder(StringBuilder stringBuilder,
+                                     String text, int number) {
+        stringBuilder.append(text).append(number).append(System.lineSeparator());
+    }
 
+    private List<String> readFromFile(String fromFileName) {
+        File file = new File(fromFileName);
+        Stream<String> lines = null;
```

Comment:
You've declared a Stream variable here but you're not closing it at any point. This might lead to the resource leak. It's better to use try-with-resources statement to automatically close the Stream.